_observing-your-data/alerting/comp-monitors.md

```diff
@@ -0,0 +1,633 @@
+            "name": "jnkjn",
+            "triggered": true,
+            "action_results": {},
+            "error": null
+        },
+        "m1ORm2": {
+            "name": "jnkjn",
+            "triggered": true,
+            "action_results": {},
+            "error": null
+        }
+    },
+    "monitor_run_results": [{
+            "monitor_name": "test triggers",
+            "period_start": 1685650668501,
+            "period_end": 1685650728501,
+            "error": null,
+            "input_results": {
+                "results": [{
+                    "bhjh": [
+                        "OkGceIgBYKBG2nHoyHAn|test1",
+                        "O0GceIgBYKBG2nHozHCW|test1"
+                    ],
+                    "nkjkj": [
+                        "OkGceIgBYKBG2nHoyHAn|test1",
+                        "O0GceIgBYKBG2nHozHCW|test1"
+                    ],
+                    "jknkjn": [
+                        "OkGceIgBYKBG2nHoyHAn|test1",
+                        "O0GceIgBYKBG2nHozHCW|test1"
+                    ]
+                }],
+                "error": null
+            },
+            "trigger_results": {
+                "NC3Dd4cBCDCIfBYtViLI": {
+                    "name": "njkkj",
+                    "triggeredDocs": [
+                        "OkGceIgBYKBG2nHoyHAn|test1",
+                        "O0GceIgBYKBG2nHozHCW|test1"
+                    ],
+                    "action_results": {},
+                    "error": null
+                }
+            }
+        },
+        {
+            "monitor_name": "test triggers 2",
+            "period_start": 1685650668501,
+            "period_end": 1685650728501,
+            "error": null,
+            "input_results": {
+                "results": [{
+                    "bhjh": [
+                        "PEGceIgBYKBG2nHo1HCw|test",
+                        "PUGceIgBYKBG2nHo3HA8|test"
+                    ],
+                    "nkjkj": [
+                        "PEGceIgBYKBG2nHo1HCw|test",
+                        "PUGceIgBYKBG2nHo3HA8|test"
+                    ],
+                    "jknkjn": [
+                        "PEGceIgBYKBG2nHo1HCw|test",
+                        "PUGceIgBYKBG2nHo3HA8|test"
+                    ]
+                }],
+                "error": null
+            },
+            "trigger_results": {
+                "NC3Dd4cBCDCIfBYtViLI": {
+                    "name": "njkkj",
+                    "triggeredDocs": [
+                        "PEGceIgBYKBG2nHo1HCw|test",
+                        "PUGceIgBYKBG2nHo3HA8|test"
+                    ],
+                    "action_results": {},
+                    "error": null
+                }
+            }
+        }
+    ],
+    "execution_start_time": "2023-06-01T20:18:48.511874Z",
+    "execution_end_time": "2023-06-01T20:18:53.682405Z",
+    "error": null
+}
+```
+
+
+### Acknowledge Chained Alerts
+
+This API provides a list of chained alerts belonging to a specific workflow. 
```

Comment:
API description:
After getting your chained alerts you can acknowledge any number of active chained alerts in one call. If the alert is already in an ERROR, COMPLETED, or ACKNOWLEDGED state, it appears in the failed array.